src/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,27 @@
+import Popup from 'reactjs-popup';
+import 'reactjs-popup/dist/index.css';
+import Style from './Popup.module.scss';
+import Image from 'next/image';
+
+export default function Modal({
+  isOpen,
+  errorText,
+}: {
+  isOpen: boolean;
+  errorText: string;
+}) {
+  return (
+    <Popup
+      open={isOpen}
+      modal
+      closeOnDocumentClick
+      contentStyle={{ backgroundColor: 'brown' }}>
```

Comment:
You can overwrite popup css classes, so never write inline styles. If you need to put some styles from react to css use css variables https://react-popup.elazizi.com/css-styling/